### ISSUE TITLE
Add DeleteTag, DeleteBranch, CreateTag, and CreateBranch functions

### ISSUE THREAD

**Issue by DataDavD** [ktrysmt/go-bitbucket#154]
I noticed that `go-bitbucket` doesn't have functions/methods for Deleting Tags/Branches and Creating Tags/Branches. I wanted to work on adding these functions after the PR for [issue 153](https://github.com/ktrysmt/go-bitbucket/issues/153) is complete.

If you approve/agree for this change, then can you please assign this issue to me @ktrysmt, please. Like issue 153 I've already started working on this in my fork.

Thanks in advance.

Best,
dd

**Comment by ktrysmt** [comment]
That's right and thank you. This doesn't have these. I'll wait your contribution! @DataDavD

**Comment by DataDavD** [comment]
Ok great. Thank you very much @ktrysmt. Can you please assign me to this issue.

Thank you!!!

Best,
dd
